src/components/Form/Form.js

```diff
@@ -1,83 +1,99 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 import uniqId from 'uniqid';
 import './Form.scss';
-import { Button } from '../Button/Button';
+import {Button} from '../Button/Button';
 
 const CN = 'custom-form';
 
 export class Form extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      title: '',
-      text: '',
-      id: uniqId()
-    };
-  }
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: '',
+            text: '',
+            id: uniqId()
+        };
 
-  static getDerivedStateFromProps(nextProps, prevState) {
-    if (nextProps.post.id !== prevState.id) {
-      return {
-        id: nextProps.post.id,
-        text: nextProps.post.text,
-        title: nextProps.post.title
-      };
     }
-    return null;
-  }
 
-  onLabelChange = e => {
-    const { id } = e.target;
-    this.setState({
-      [id]: e.target.value
-    });
-  };
+    onLabelChange = (event) => {
+        const {id, value} = event.target;
+
+        this.setState({
+            [id]: this.state[id].length < 1 ? value.trim() : value
```

Comment:
Why do you need this condition `this.state[id].length < 1`? what's the logic behind?

Reply:
I thought this condition is necessary if the user click on 'Space' in forms and I had no idea of solving that better.

Reply:
@AlyonaPianykh I thought this condition is necessary if the user click on 'Space' in forms and I had no idea of solving that better.